ush/run_verif_global_in_global_workflow.sh

```diff
@@ -49,8 +44,8 @@ export model_hpss_dir_list=${model_hpss_dir:-/NCEPDEV/$HPSS_PROJECT/1year/$USER/
 export model_data_run_hpss=${get_data_from_hpss:-"NO"}
 export hpss_walltime=${hpss_walltime:-10}
 ## DATE SETTINGS
-export start_date="$VDATE"
-export end_date="$VDATE"
+export start_date="${vPDY}"
+export end_date="${vPDY}"
```

Comment:
```suggestion
export start_date="${VDATE}"
export end_date="${VDATE}"
```
`VDATE` is set as `YYYYMMDDHH` where as `vPDY` is `YYYYMMDD` and does not contain the `HH`.
I suspect this needs to stay as `VDATE`.  Same for some of the changes below.
Will make suggestions as I review.

Reply:
My bad!  I see `VDATE` is being cut to `YYYYMMDD`